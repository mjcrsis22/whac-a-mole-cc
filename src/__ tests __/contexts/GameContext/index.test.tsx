import { render } from '@testing-library/react';
import GameContextProvider, {
  GameContext,
} from '../../../contexts/GameContext';
import { useContext } from 'react';

const GameContextConsumer = () => {
  const { state } = useContext(GameContext);
  return <>{state.started ? 'Started' : 'Stopped'}</>;
};

describe('GameContext', () => {
  test('Provides data', () => {
    const { container } = render(
      <GameContextProvider>
        <GameContextConsumer />
      </GameContextProvider>,
    );
    expect(container).toMatchSnapshot();
  });
});
