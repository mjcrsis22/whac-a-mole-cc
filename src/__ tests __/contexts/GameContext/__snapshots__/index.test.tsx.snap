// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameContext Provides data 1`] = `
<div>
  Stopped
</div>
`;
